perllib/FixMyStreet/Cobrand/Brent.pm

```diff
@@ -302,6 +304,15 @@ Same as Symology above, but different attribute name.
                 $row->update_extra_field({ name => 'usrn', description => 'USRN', value => $ref });
             }
         }
+
+=item * Adds location name from WFS service for ATAK reports if missing
+
+=cut
+
+    } elsif ($contact->email =~ /^ATAK/) {
+        if (my $name = $self->lookup_location_name($row)) {
```

Comment:
Comment says only do if missing, but it's always doing the lookup and replacing what's already there

Reply:
Ooops, thanks! Have added a check so to avoid the extra HTTP request if it's not necessary (along with a test) in https://github.com/mysociety/fixmystreet/pull/4551/commits/14d85f56fc20a3d064e51f8bc61ac2ccfcd24e40.

Also removed the fixtures in https://github.com/mysociety/fixmystreet/pull/4551/commits/a2532b3a4c71f7f4ed4b0e047c35f3eaa7a20b1d and fixed the tests in https://github.com/mysociety/fixmystreet/pull/4551/commits/20098d9686c14eb6b16de1b4f501ab891b922772.